lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -532,10 +653,16 @@ func (c *ContainerProxy) Signal(vc *viccontainer.VicContainer, sig uint64) error
 			}
 		}
 
-		c.Wait(vc, -1*time.Second)
+		// Wait awhile to see if signal killed the container
+		if _, _, _, err := c.Wait(vc, killWaitBeforeForce); err != nil {
```

Comment:
The distinction between stop and kill in VIC is very thin.  Stop in VIC causes a VM poweroff.  This is actually what we want if signaling fails.  Regular docker will kill the process if signaling the container fails.  Kill is kind of a definitive shutdown and not a nice shutdown.  This is why I don't think there is a need to retry.  It just so happens in VIC, Stop does the same thing.

Added a default case.